### ISSUE TITLE
Bind Parameter(s) dialog

### ISSUE THREAD

**Issue by qiqingli** [dbeaver/dbeaver#18776]
### Description

https://user-images.githubusercontent.com/20805098/212645847-9f09de55-1c51-427c-adef-35d738b304a9.mp4

### DBeaver Version

Community Edition 22.3.1

### Operating System

macos Ventura 13.1

**Comment by E1izabeth** [comment]
Can't reproduce on Windows 10.

**Comment by qiqingli** [comment]
> Can't reproduce on Windows 10.

This problem occurs on macOS 13.1.

**Comment by qiqingli** [comment]
There is another problem on macOS 13.1. The Bind Paramter(s) dialog of DBeaver22.3.2 version capitalizes all the binding parameters.
<img width="516" alt="image" src="https://user-images.githubusercontent.com/20805098/212790528-31e579fd-553a-4d7c-b883-b76ca9a634bf.png">

**Comment by E1izabeth** [comment]
Capitalisation was an intended change. Parameters names always was case-insensitive and capitalization just highlights it.
We have added support of quotation of parameters names to add possibility to make them case-sensitive.

**Comment by qiqingli** [comment]
> Capitalisation was an intended change. Parameters names always was case-insensitive and capitalization just highlights it. We have added support of quotation of parameters names to add possibility to make them case-sensitive.

I don't think this is a good change.
It may be a good alternative to provide case-sensitive configuration of binding parameters in Preferences settings.
<img width="929" alt="image" src="https://user-images.githubusercontent.com/20805098/212795631-8bf403a5-0e68-41ba-8f3a-f9fbd7c6da1c.png">

**Comment by jcachat** [comment]
I agree, this is a terrible change.  It broke all of my scripts.  I now have to change all of the "@set" commands from something like "@set state='SC'" to '@set STATE='SC'" even though I can reference those variables in the sql scripts as ":state".

I don't understand from your comment if you are intending this to be case sensitive or not but right not, it's just wrong.  The only way I can get it to work is to capitalize the @set commands and then it seems to be insensitive when I reference the variables.

**Comment by E1izabeth** [comment]
@jcachat you are saying about the bug #18706
The fix is available in the early access version (https://dbeaver.io/files/ea/) and will be available in the next release.

**Comment by jcachat** [comment]
Yes, sorry - I didn't see that when I was searching since it's closed already.  Thanks!

**Comment by arhayka** [comment]
Hi @qiqingli,
Thank you for your report.
I just reproduced this issue on macOS on DBeaver 22.3.5
